app/lib/webfinger_resource.rb

```diff
@@ -0,0 +1,60 @@
+class WebfingerResource
+  attr_reader :resource
+
+  def initialize(resource)
+    @resource = resource
+  end
+
+  def username
+    case resource
+    when /https/
```

Comment:
Ah I missed that in the original ... so both of these strings should work?

- `http://example.com/users/name` -> `name`
- `https://example.com/users/name` -> `name`

?